marklogic-data-hub/build.gradle

```diff
@@ -154,13 +132,15 @@ build.dependsOn clientJar
 
 node {
   // Version of node to use.
-  version = '8.10.0'
+  version = '16.15.0'
```

Comment:
@ryanjdew  Do we need to bump up node version too ? On hub-central we are using node = '14.15.4'
  and  npmVersion = '6.14.10'

Reply:
I'll change it to bt the same versions as Hub Central